styleq.flow.js

```diff
@@ -7,15 +7,15 @@
  * @flow strict
  */
 
-export type CompiledStyle = {
+export type CompiledStyle = $ReadOnly<{
   $$css: true,
   [key: string]: string,
-};
+}>;
 
-export type InlineStyle = {
+export type InlineStyle = $ReadOnly<{
   $$css?: empty,
-  [key: string]: number | string,
-};
+  [key: string]: mixed,
```

Comment:
Why does the value need to change to `mixed`?

Reply:
I did this to account for things like `Animated` values. It's not _strictly_ neccessary. 

I wanted to put this up and ask about whether I should set up the scripts to generate Flow and TS type defs. Then I can update the Flow version as well.

Reply:
On web, we're not going to get Animated values

Why do we need a script to generate type defs instead of just manually managing the really small def files that already exist?

Reply:
I can do it manually too. A script ensures we don't accidentally write the wrong types, but it is very small, so I can do it manually.